tests/analog/message.test.ts

```diff
@@ -430,6 +680,111 @@ const validUplinkMessages: TMessageExamples = {
     }
 };
 
+const validMultichannelUplinkMessages: TMessageExamples = {
+    'multichannel hourMc + lastEvent': {
+        bytes: getBytesFromHex(
+            '17 33 00 68 ec 0f ad b0 48 c6 08 c8 08 c6 08 c4 08 be 08 b4 08 b9 08 b3 ed 2d 00 00 00 00'
+            + ' 00 00 00 8d b0 5a 00 00 00 00 00 00 00 8f de 2a 00 00 00 00 00 00 00 63 39 80 00 ad'
+        ),
+        commands: [
+            {
+                id: uplinkCommands.hourMc.id,
+                name: 'hourMc',
+                headerSize: 2,
+                config: {
+                    hardwareType: hardwareTypes.IMP4EU
+                },
+                parameters: {
+                    startTime2000: 5832000,
+                    hours: 8,
+                    channelList: [
+                        {
+                            value: 1185837,
+                            diff: [
```

Comment:
can we write all numbers in a single line?

and below

Reply:
fixed